src/content/docs/en/guides/backend/supabase.mdx

```diff
@@ -122,7 +122,7 @@ To add authentication to your project, you will need to create a few server endp
 - `POST /api/auth/signin`: to sign in a user.
 - `GET /api/auth/signout`: to sign out a user.
 
-Create these endpoints in the `src/pages/api/auth` directory of your project. Your project should now include these new files:
+Create these endpoints in the `src/pages/api/auth` directory of your project. Make sure to specify `export const prerender = false` to render the endpoints on demand. Your project should now include these new files:
```

Comment:
```suggestion
Create these endpoints in the `src/pages/api/auth` directory of your project. If you are using `hybrid` rendering mode, you must specify `export const prerender = false` at the top of each file to render these endpoints on demand. Your project should now include these new files:
```
This is only for `hybrid` mode, not `server` mode, so we we should make that clear!